include/seqan/misc/bit_twiddling.h

```diff
@@ -514,7 +514,7 @@ _bitScanForward(TWord word, WordSize_<NUM_BITS>)
     return _bitScanForwardGeneric(word, WordSize_<NUM_BITS>());
 }
 
-#if defined(PLATFORM_GCC)
+#if defined(COMPILER_GCC) || defined(COMPILER_INTEL) || defined(COMPILER_CLANG)
```

Comment:
Whats this __builtin_ctzll stuff? Isnt that the reverse of popcount? Does that no exist on windows? Are we sure it exists on Intel et cetera?